README.md

```diff
@@ -1,13 +1,27 @@
 # OnlineChess
 
-Client/server базирано приложение за игра на шах. Очаквана функционалност:
+Client/server based web api for playing chess. Describtion of the functionality:
 
-    -Стая с всички потребители, които са налиния в момента.
-
-    -Покани за игра.
-    
-    -Играта да познава всички правила на шаха и да не позволява невалидни ходове.
-    
-    -Не се очаква AI/алгоритъм, който да играе автоматично.
-
-    -Web client.
\ No newline at end of file
+This project represents a web site to play
+some online chess. It is mobile friendly 
+and can be played from a phone. The 
+connection is established by web sockets 
+and can be played on a LAN connection. 
+This site is basicaly an advanced chat 
+where instead of messages the players send 
+each other notation for the next move. 
+After each move the current board 
+placement is showed on the display. There 
+are private rooms for the players so that 
+when you find a match noone will bother 
```

Comment:
No one not noone, but everything else look nice. Overall you did a pretty good job on the README.